starknet_py/tests/e2e/client/gateway_test.py

```diff
@@ -144,45 +144,24 @@ async def test_get_storage_at_incorrect_address_gateway_client(gateway_client):
 
 
 @pytest.mark.asyncio
-async def test_get_l1_handler_transaction(gateway_client):
-    with patch(
-        "starknet_py.net.http_client.GatewayHttpClient.call", AsyncMock()
-    ) as mocked_transaction_call:
-        mocked_transaction_call.return_value = {
-            "status": "ACCEPTED_ON_L1",
-            "block_hash": "0x38ce7678420eaff5cd62597643ca515d0887579a8be69563067fe79a624592b",
-            "block_number": 370459,
-            "transaction_index": 9,
-            "transaction": {
-                "version": "0x0",
-                "contract_address": "0x278f24c3e74cbf7a375ec099df306289beb0605a346277d200b791a7f811a19",
-                "entry_point_selector": "0x2d757788a8d8d6f21d1cd40bce38a8222d70654214e96ff95d8086e684fbee5",
-                "nonce": "0x34c20",
-                "calldata": [
-                    "0xd8beaa22894cd33f24075459cfba287a10a104e4",
-                    "0x3f9c67ef1d31e24b386184b4ede63a869c4659de093ef437ee235cae4daf2be",
-                    "0x3635c9adc5dea00000",
-                    "0x0",
-                    "0x7cb4539b69a2371f75d21160026b76a7a7c1cacb",
-                ],
-                "transaction_hash": "0x7e1ed66dbccf915857c6367fc641c24292c063e54a5dd55947c2d958d94e1a9",
-                "type": "L1_HANDLER",
-            },
-        }
-
-        transaction = await gateway_client.get_transaction(tx_hash=0x1)
+async def test_get_deploy_account_transaction(
+    gateway_client, deploy_account_transaction_hash
+):
+    # TODO Modify this test to also use FullNodeClient (move to client_test)
```

Comment:
Do we have an issue for this? If not, let's create one.

Reply:
https://github.com/software-mansion/starknet.py/issues/418

I've added a checkbox